Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -330,6 +337,11 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
 
         this._currentSessionOrGroup = savedCurrentConsoleGroup;
 
+        if (this._showTimestamps)
+            this._currentSessionOrGroup.element.removeAttribute("hide-timestamps");
+        else
+            this._currentSessionOrGroup.element.setAttribute("hide-timestamps", "true");
+
```

Comment:
In new code we generally prefer using a CSS class for this, which also allows this to be reduced to a single line.
```
this._currentSessionOrGroup.element.classList.toggle("timestamps-visible", this._showTimestamps);
```